src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -238,6 +240,41 @@ get_schemas_ids_id(server::request_t rq, server::reply_t rp) {
     co_return rp;
 }
 
+struct schema_version_json : public ss::json::json_base {
+    ss::json::json_element<ss::sstring> subject;
```

Comment:
It's a shame this can't be a `std::string_view`, but I'm not sure it's worth fixing for now.

Reply:
yeah...this usage pattern is endemic to types derived from `ss::json::json_base` (for our purposes, mostly generated code) fwiw. 

I'd probably prefer a move-assignable `json_element` anyway.